LoRaEngine/deployment.ci.lbs.template.json

```diff
@@ -102,6 +102,31 @@
             "version": "1.0",
             "status": "running",
             "restartPolicy": "always"
+          },
+          "metricscollector": {
```

Comment:
If we set that on EFLOW and the AIO gateway, why do we need it again in the LBS template?

Reply:
Imho it helps keep a baseline with a deployment not deploying everything